index.js

```diff
@@ -444,6 +449,7 @@ function fastifyView (fastify, opts, next) {
         }
         const html = engine.render(templateString, data, partialsObject)
 
+        /* istanbul ignore else */
```

Comment:
It would be good to remove these ignores

Reply:
> Good work! This is solid. Is there anything more you'd like to do here?

Not sure if there is anything else.Maybe something you would like to propose else I can remove the ignores and squash everything to one commit , is that fine ?